docs/articles_en/openvino-workflow/running-inference/string-tensors.rst

```diff
@@ -203,4 +203,4 @@ Additional Resources
 
 * Use `OpenVINO tokenizers <https://github.com/openvinotoolkit/openvino_contrib/tree/releases/2024/0/modules/custom_operations/user_ie_extensions/tokenizer/python>`__ to produce models that use string tensors to work with textual information as pre- and post-processing for the large language models.
 
-* Check out `GenAI Samples <https://github.com/openvinotoolkit/openvino.genai/tree/master/samples/cpp/greedy_causal_lm>`__ to see how string tensors are used in real-life applications.
+* Check out `GenAI Samples <https://github.com/openvinotoolkit/openvino.genai/tree/releases/2024/6/samples/cpp/greedy_causal_lm>`__ to see how string tensors are used in real-life applications.
```

Comment:
this link is not valid, the correct one is https://github.com/openvinotoolkit/openvino.genai/tree/releases/2024/6/samples/cpp/text_generation (the same in some other places)

Reply:
We'll go through sample links anyway, once all the intermediate PRs are merged. We'll try to align all of them before updating the website.